### ISSUE TITLE
pip install bitmerchant no lnoger works without a virtualenv

### ISSUE THREAD

**Issue by mflaxman** [sbuss/bitmerchant#48]
```
$ pip install bitmerchant
Downloading/unpacking bitmerchant
  Running setup.py egg_info for package bitmerchant
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "/var/folders/t4/43065rvd4gx9rmjjs38g_0z80000gn/T/pip-build/bitmerchant/setup.py", line 3, in <module>
        from pip.download import PipSession
    ImportError: cannot import name PipSession
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "/var/folders/t4/43065rvd4gx9rmjjs38g_0z80000gn/T/pip-build/bitmerchant/setup.py", line 3, in <module>

    from pip.download import PipSession

ImportError: cannot import name PipSession

----------------------------------------
Command python setup.py egg_info failed with error code 1 in /var/folders/t4/43065rvd4gx9rmjjs38g_0z80000gn/T/pip-build/bitmerchant
Storing complete log in /Users/foo/.pip/pip.log

```

It's because of this recent changed you introduced @sbuss : https://github.com/sbuss/bitmerchant/commit/8873fb4f1a83824c6d20d48a6fe22404902f8c36#diff-2eeaed663bd0d25b7e608891384b7298R3

If you first create and activate a virtualenv, then it works as expected. Of course that's the best practice, but it should work in all cases.

**Comment by sbuss** [comment]
Sorry about that @mflaxman, I do all development in virtualenvs so I didn't notice it. What version of `pip` are you using?

**Comment by mflaxman** [comment]
```
$ pip --version
pip 1.2.1 from /usr/local/lib/python2.7/site-packages/pip-1.2.1-py2.7.egg (python 2.7)
```

I setup this OSX machine 2+ years ago, I don't remember if it came pre-installed or if I installed pip myself.

**Comment by sbuss** [comment]
That's a pretty old version of pip. But I'm just going to remove the offending lines. It's bad practice to import things outside of stdlib in setup.py, anyway. If #49 passes tests then I'll get a new version out tonight.

**Comment by sbuss** [comment]
New release is tagged. Travis-ci should push it to pypi within the hour.

**Comment by sbuss** [comment]
Fixed by #49
